sqlglot/expressions.py

```diff
@@ -4168,6 +4168,11 @@ class Abs(Func):
     pass
 
 
+class Flatten(Func):
+    arg_types = {"expressions": True}
+    is_var_len_args = True
```

Comment:
This is not used right?

Reply:
delete it and merge

Reply:
I see, it was added to represent Spark's `Flatten` judging from the dataframe test. I'll merge and delete it since we don't use it elsewhere.

Reply:
i thought you meant just the is_var_len_args